src/handlers/motions/helpers.ts

```diff
@@ -18,3 +19,43 @@ export const updateMotionInDB = async (
     },
   });
 };
+
+/*
+ * Motion ids are not unique between voting reputation installations. If you had a motion with id of 1
+ * in a previous voting reputation installation, you will get another one after uninstalling and reinstalling
+ * the voting reputation client. So, we only want the one created most recently, since that's the one created
+ * by the currently installed version of the voting reputation extension.
+ */
+
+export const getMotionFromDB = async (
+  colonyAddress: string,
+  motionId: BigNumber,
+): Promise<MotionQuery | undefined> => {
+  const { items: motions }: { items: MotionQuery[] } = await query(
```

Comment:
You could make `query` accept a generic that would describe the shape of the response